src/backend/pg.rs

```diff
@@ -96,9 +96,9 @@ impl SqlGenerator for Pg {
     fn create_index(table: &str, schema: Option<&str>, name: &str, _type: &Type) -> String {
         // FIXME: Implement PG specific index builder here
         format!(
-            "CREATE {} INDEX \"{}\" ON {}\"{}\" ({})",
+            "CREATE {}INDEX \"{}\" ON {}\"{}\" ({})",
             match _type.unique {
-                true => "UNIQUE",
+                true => "UNIQUE ",
```

Comment:
I don't like this change. I'd rather keep padding in the format string, than the interpolators. I can't imagine `mysql` would have an issue with multiple spaces in an Sql command?

Reply:
@spacekookie OK I can revert this change. I should say though that the change isn't for stylistic reasons, but rather practically motivated since when debugging and reading the generated SQL the extra space looks weird/out of place.